examples/hello-world/step-by-step/readme.md

```diff
@@ -0,0 +1,52 @@
+#  Step-by-Step Examples
+
+When give an machine learning problem, we probably wonder, 
+where do we start to formulate the federated learning problem. 
+
+* What the data looks like ?
+* How do we analyze the data and compare global statistics with site's data analytics ? 
```

Comment:
How do we analyze the data and compare global statistics with the site's data analytics?